ci-operator/step-registry/workers-rhel/upgrade/workers-rhel-upgrade-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: workers-rhel-upgrade
+  steps:
+  - ref: workers-rhel-upgrade-repos
```

Comment:
If the step `workers-rhel-upgrade-repos` is the same as `workers-rhel-repos`, could we use `workers-rhel-repos` directly here?

Reply:
Prow doesn't support to use a step multiple times in a test or workflow. So we have to make a copy and rename it.

Reply:
Can we simply skip this step as it's already contained in another one ?